### DISCUSSION TITLE
scheduled functions local dev experience improvements

### DISCUSSION THREAD

**Discussion by brianleroux** [architect/architect#1276] [Feature Requests]
> Thinking back to my scraping days and I would have to re-run jobs a lot, and I had a dashboard that would manually run Cron jobs

super good ideas from @wesbos

- [ ] Some command or UI where I could manually trigger them 
- [ ] URLs for development mode where we could ping to trigger them `localhost:3333:/___scheduled/functionName`
- [ ] support .env and we can throw require(‘dotenv’).config() at the top of the script and manually run them outside arc

**Comment by filmaj** [comment]
FYI someone issued a PR to start on some of this work a couple months back: https://github.com/architect/sandbox/pull/466
